org.openhab.binding.zigbee/README.md

```diff
@@ -129,6 +145,7 @@ The following coordinators are known to be supported.
 |----------------------------|-------------|-------------------------------|
 |[Texas Instruments CC2531EMK](http://www.ti.com/tool/cc2531emk)|[CC2531](#cc2531-coordinator)|Needs extra hardware and correct firmware (might be hard to find) for flashing.<br>There are also cheap copies of the CC2531 Stick available on eBay, Aliexpress, etc. like [this](https://de.aliexpress.com/item/Drahtlose-Zigbee-CC2531-Sniffer-software-protokoll-analyse-Bareboard-Paket-Protokoll-Analyzer-Modul-Usb-schnittstelle-Dongle-Erfassen/32852226435.html) and a module to flash the firmware like [this](https://de.aliexpress.com/item/SmartRF04EB-CC1110-CC2530-ZigBee-Module-USB-Downloader-Emulator-MCU-M100/32673666126.html) including a [connector board](https://de.aliexpress.com/item/CC2531-CC2540-Zigbee-Sniffer-software-protokoll-analyse-Wireless-Board-Bluetooth-BLE-4-0-Dongle-Capture-Modul/32869263224.html)<br>Also CC2530, CC2538 or CC2650 may work with the correct firmware but are not suggested|
 |[Bitron Video ZigBee USB Funkstick](http://www.bitronvideo.eu/index.php/produkte/smart-home-produkte/zb-funkstick/)|[Ember](#ember-ezsp-ncp-coordinator)| |
+|[Elelabs ELU013/ELR023](https://elelabs.com/shop/)|[Ember](#ember-ezsp-ncp-coordinator)| Both the stick and the hat can be upgraded without additional hardware, firmware available [here](https://github.com/Elelabs/elelabs-zigbee-ezsp-utility). Use baud rate 115200 and hardware flow control. |
```

Comment:
https://community.openhab.org/t/eu-zigbee-dongle/31069/163

All of them, or just the ones based on EM358/EFR32?

Also, as this is an Ember-specific configuration, might be worth moving the setting to the `Ember NCP` section.

Reply:
I'm flexible - I was initially thinking a table showing all the coordinators supported, and the settings for each one might be a good approach? I'm not sure how that would look - maybe it's too much information in one table, but I think it would be clearer? I've not tried it though, so I'm happy for you to suggest something.

While this is an Ember thing, and at this stage I don't think it depends on the actual stick, I think it might still be clearer to simply list each stick. I know that repeats information, but it's probably also clearer? Again - I'm just thinking out loud a bit here so if you propose something that looks good, doesn't spread the information too far and wide, then I'll probably be fine with it...